src/test/java/bio/terra/dao/DatasetDaoTest.java

```diff
@@ -159,7 +161,11 @@ public void datasetEnumerateTest() throws Exception {
         String datasetName = datasetRequest.getName() + UUID.randomUUID().toString();
 
         for (int i = 0; i < 6; i++) {
-            datasetRequest.name(makeName(datasetName, i));
+            datasetRequest
+                .name(makeName(datasetName, i))
+                // set the description to a random string so we can verify the sorting is working independently of the
+                // study name or created_date. add a suffix to filter on for the even datasets
+                .description(UUID.randomUUID().toString() + ((i % 2 == 0) ? "__foo__" : ""));
```

Comment:
The __foo__ would match ABfooCD

Reply:
I believe these underscores will be exact matched, they will end up in the query as `WHERE name ILIKE '%__foo__%'`

Reply:
Postgres docs say that `_` is the single character match in LIKE.
Why do you think they will be exact matched?